### DISCUSSION TITLE
Works on ThinkPad X280! 🙌👏👏

### DISCUSSION THREAD

**Discussion by Real0ver** [valnoxy/t480-oc#47] [Show and tell]
Works on my ThinkPad X280 normally on macOS Monterey!

Hardware:
Model: Lenovo ThinkPad X280
UEFI Version: N20ET62W 1.47
CPU: Intel(R) Core(R) i5-8350U @ 1.7GHz (turbo boost 3.6GHz)
iGPU: Intel UHD Graphics 620
RAM: 8GB DDR4 Dual Channel RAM (soldered to motherboard)
SSD: SUPERSONIC 512GB
Wi-Fi & Bluetooth Adapter: Intel(R) Dual Band Wireless-AC 8265 (802.11a/b/g/n/ac)
Has fingerprint reader
Display: 1080p 60Hz IPS Laptop display
2 Thunderbolt 3 USB-C Ports
LAN: Intel(R) Ethernet Connection (4) I219-LM
Input: Synaptics SMBus Touchpad, PS/2 Keyboard (JP Layout)

**Comment by jim040296** [comment]
can i have your config.plist sir cause i'm struggling to activate the wifi card
